Patches/Vanilla Apparel Expanded/ThingDefs_Misc/Apparel_Industrial.xml

```diff
@@ -1,121 +1,176 @@
+					<value>
+						<Cloth>20</Cloth>
+					</value>
+				</li>
 
-			<!-- == VAE_Apparel_BuildersJacket == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Apparel_BuildersJacket"]/statBases/ArmorRating_Sharp</xpath>
-				<value>
-					<Bulk>5</Bulk>
-					<WornBulk>1</WornBulk>
-					<ArmorRating_Sharp>0.55</ArmorRating_Sharp>
-				</value>
-			</li>
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Apparel_BuildersJacket"]/statBases/ArmorRating_Blunt</xpath>
-				<value>
-					<ArmorRating_Blunt>0.4</ArmorRating_Blunt>
-				</value>
-			</li>
+				<!-- == VAE_Apparel_DoctorScrubs == -->
+				<!-- cloth cost -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_DoctorScrubs"]/costList/Cloth</xpath>
+					<value>
+						<Cloth>30</Cloth>
+					</value>
+				</li>
 
-			<!-- == VAE_Apparel_LabCoat == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Apparel_LabCoat"]/statBases/ArmorRating_Sharp</xpath>
-				<value>
-					<Bulk>7.5</Bulk>
-					<WornBulk>1</WornBulk>
-					<ArmorRating_Sharp>0.1</ArmorRating_Sharp>
-				</value>
-			</li>
+				<!-- == VAE_Apparel_LabCoat == -->
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_LabCoat"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<Bulk>7.5</Bulk>
+						<WornBulk>1</WornBulk>
+						<ArmorRating_Sharp>0.1</ArmorRating_Sharp>
+					</value>
+				</li>
+				<!-- cloth cost -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_LabCoat"]/costList/Cloth</xpath>
+					<value>
+						<Cloth>40</Cloth>
+					</value>
+				</li>
 
-			<!-- == VAE_Apparel_SheriffShirt == -->
-			<!-- statBases -->
-			<li Class="PatchOperationReplace">
-				<xpath>Defs/ThingDef[defName="VAE_Apparel_SheriffShirt"]/statBases/StuffEffectMultiplierArmor</xpath>
-				<value>
-					<StuffEffectMultiplierArmor>1.1</StuffEffectMultiplierArmor>
-				</value>
-			</li>
-		</operations>
+				<!-- == VAE_Apparel_SheriffShirt == -->
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_SheriffShirt"]/statBases/StuffEffectMultiplierArmor</xpath>
+					<value>
+						<StuffEffectMultiplierArmor>1.1</StuffEffectMultiplierArmor>
+					</value>
+				</li>
+
+				<!-- == costStuffCounts == -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_TankTop"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>15</costStuffCount>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_CasualTShirt" or defName="VAE_Apparel_Shorts" or defName="VAE_Apparel_Skirt" or defName="VAE_Apparel_Trousers"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>20</costStuffCount>
```

Comment:
This is a little too cheap for the Trousers. They're a pair of dress pants, versus the regular pants which I see more as a pair of sweatpants. The Trousers are a little heavier, and need a bit more material. 30, perhaps?

Reply:
I upped them to 25 instead, as the jeans cost 30 and the jeans are more thick than trousers.